app.rb

```diff
@@ -0,0 +1,23 @@
+require_relative "calc.rb"
+#'run calculation' variable
+run_calc = 1
+
+# "run calculation" conditions
+
+while run_calc == 1 do 
+	puts "Please select calculator you would like to use? (1)Basic (2)Advanced (3)BMI"
+
+	calc_type = gets.to_i
+
+	answer = Calculator.new
```

Comment:
You create a new Calculator object here, but you don't pass it any arguments - in your initialize function for the Calculator class, you say that it must take one argument: "answer".